docs/api-status.json

```diff
@@ -494,6 +494,26 @@
       {
         "name": "FSAdmin.VolumeStatus",
         "comment": "VolumeStatus returns a VolumeStatus object for the given volume name.\n\nSimilar To:\n ceph fs status cephfs <name>\n"
+      },
```

Comment:
`make api-update` didn't work for me :-(
Had manually edited this with an online json tool.

Reply:
Actually, I can not reproduce the CI failure locally. So I'm a bit unclear on what's actually going on. Please hold off working on this until I report back. Sorry for the noise.

Reply:
OK, after one SNAFU where I was trying the changes from your other PR, I think the proper and simplest fix is to simply drop the 2nd patch in your pr ('docs: update subvolume metadata API details'). Because the code is now also guarded  by the `ceph_ci_untested` tag there's no need to do the API stuff until it is in proper `preview` status, eg. when it's only guarded by the `ceph_preview` build tag.

I've actually tried this locally and it worked so I have high confidence in this suggestion, and it should be pretty easy to implement on your side. :-)
Just remember when we come from untested status to preview status, then we'll need to do the api status tracking json changes. Hopefully then we can work out any issues you have with the `make api-update` command and so it'll mostly be automatic at that time.

Reply:
I looked into this, and it seems `implements` evaluates the tags itself. Since `ceph_ci_untested` is not set it ignores these files. So everything works as designed. You should simply not touch this file in this PR and call `make api-update` later after you removed the untested tag in a later PR.

Reply:
Sure @phlogistonjohn and @ansiwen, dropped the 2nd patch for now. Thanks!